connectivity/check/action.go

```diff
@@ -203,7 +207,7 @@ func (a *Action) ExecInPod(ctx context.Context, cmd []string) {
 			a.test.Log(strings.TrimSpace(stderr.String()))
 			a.test.Log()
 		} else {
-			a.test.Debugf("command %q failed as expected: %s", cmdStr, err)
+			a.test.Debugf("command %q failed as expected: %s:\n%s", cmdStr, err, stderr.String())
```

Comment:
Nit: `%s` should already call `.String()` on a parameter if it implements Stringer.

Reply:
fixed :-)